frontend/test/metabase/scenarios/admin/databases/add.cy.spec.js

```diff
@@ -52,6 +53,28 @@ describe("scenarios > admin > databases > add", () => {
     cy.url().should("match", /\/admin\/databases\?created=42$/);
   });
 
+  it("should trim fields needed to connect to the database", () => {
+    cy.route({
+      method: "POST",
+      url: "/api/database",
+      response: { id: 42 },
+      delay: 1000,
+    }).as("createDatabase");
+
+    cy.visit("/admin/databases/create");
+
+    typeField("Name", "Test db name");
+    typeField("Host", "localhost  \n  ");
+    typeField("Database name", " test_postgres_db");
+    typeField("Username", "   uberadmin   ");
+
+    cy.findByText("Save").click();
+
+    cy.wait("@createDatabase");
```

Comment:
I think it would make sense to assert on the request body here to ensure the whitespace was removed. There's an example of asserting on a request body a bit further down in this file:

```js
cy.wait("@createDatabase").then(({ request }) => {
    expect(request.body.engine).to.equal("postgres");
    expect(request.body.name).to.equal("Test db name");
    expect(request.body.details.user).to.equal("uberadmin");	
});	
```

Reply:
Good idea; updated the test